fillHistosScouting.py

```diff
@@ -607,12 +623,13 @@ def getSelectionSF(lxy):
                         gvec = ROOT.TLorentzVector()
                         gvec.SetPtEtaPhiM(t.GenPart_pt[k], t.GenPart_eta[k], t.GenPart_phi[k], t.GenPart_m[k])
                         dmumot.append(gvec)
-                        break
+                        break 
+            '''
             if isResonance:
```

Comment:
This wouldn't need to be commented but it is true that we are not using this information for now... So I will leave it like this and come back to it when/if neccessary